client/src/components/Options.jsx

```diff
@@ -4,13 +4,48 @@ import styled from 'styled-components';
 const OptionsContainer = styled.div`
   grid-column: 1 / 3;
   grid-row: 2 / 3;
+  border: 1px solid #5c5c5c;
+  border-right: 2px solid #5c5c5c;
+  font-family: Proxima Nova;
+  overflow: hidden;
+`;
+const Title = styled.div`
+  display: inline-block;
+  background-color: #5c5c5c;
+  font-family: Proxima Nova;
+  font-size: 13px;
+  color: #c4c4c4;
+  height: 100%;
+  width: 15%;
+  overflow: hidden;
+  text-align: center;
+  padding: 1.75vh 0.75vw 1vh 0.75vw;
+  vertical-align: top;
+  transform: scale(1 , 1.48);
+`;
+
+const Option = styled.div`
+ display: inline-block;
+ color: white;
+ margin: 1vh 0.85vw;
+ padding-top: 1px;
+ font-size: 0.85em;
+`;
+
+const Divider = styled.span`
```

Comment:
Nice, clean solution for those little dividing lines. Really impressed with the css throughout your module